CRM/Contribute/Selector/Search.php

```diff
@@ -82,6 +82,7 @@ class CRM_Contribute_Selector_Search extends CRM_Core_Selector_Base implements C
     'contribution_soft_credit_contact_id',
     'contribution_soft_credit_amount',
     'contribution_soft_credit_type',
+    'participant_contribution_id',
```

Comment:
There is no civicrm_participant_contribution table, but there is a civicrm_participant_payment table (https://github.com/civicrm/civicrm-core/blob/master/xml/schema/Event/ParticipantPayment.xml). Would this be better named participant_payment_id ?